src/contrib/dependencyinjection/Akka.DependencyInjection.Tests/ActorServiceProviderPropsWithScopesSpecs.cs

```diff
@@ -193,7 +193,6 @@ public void ActorsWithNonDiDependenciesShouldStart()
         [Fact(DisplayName = "Props created via the ServiceProvider should support the standard Props copying methods")]
         public void ServiceProvider_Props_should_support_copying()
         {
-            // <CreateNonDiActor>
```

Comment:
I see this is a duplicate tag - where is the original one used / closed? Have you had a chance to preview these changes?

Reply:
This particular tag did not have matching closing tag. I deleted that because Line 148 - 156 had opening and closing tag. Yes I tested the change locally